lib/ansible/modules/remote_management/manageiq/manageiq_user.py

```diff
@@ -183,7 +190,7 @@ def compare_user(self, user, name, group_id, password, email):
         """
         found_difference = (
             (name and user['name'] != name) or
-            (password is not None) or
+            (password is not None and self.module.params['update_password'] == 'always') or
```

Comment:
@joelddiaz ^^

a. what is the desired behavior ?
b. do want to re-set the password if the object is changed ?

if we do, @cben suggests to use a different name, from "update_password"  "on_create" it may not be clear the we cosider any chage a "creation". 

I think that the current name "update_password" is the best one, considering the options :-)
@cben can you think of better name ?